populate.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
That change would grab the first (best?) bash in the path, and the script could behave differently depending on who runs it based on current $PATH. I think it is better to be explicit and use /bin/bash directly.

Reply:
it's what is done with python though?
I personally have no trouble with that change

Reply:
If you have 2 versions of bash installed, that's probably for a reason. 

The one you want to use by default should be stored in the `SHELL` *env variable* and otherwise `/usr/bin/env bash` will point to `/bin/bash`.

If treating this as an opportunity, rather than a risk. It's up to you to decide.

Reply:
yeah, I good with that